pittapi/sports.py

```diff
@@ -26,14 +26,6 @@
 BASKETBALL_URL = "http://site.api.espn.com/apis/site/v2/sports/basketball/mens-college-basketball/teams/pittsburgh"
 
 
-class NextBasketballGame(NamedTuple):
-    name: str
-    short_name: str
-    season_name: str
-    week: int
-    court: str
-
-
 class NextFootballGame(NamedTuple):
```

Comment:
Is this NamedTuple still used? If not, we should delete this class.

Reply:
Resolved in [afdce7b](https://github.com/pittcsc/PittAPI/pull/137/commits/afdce7b22c50d0c6f18af707f379a39adcacdd4a)